### PR TITLE
Update name initial overlay

### PR DESCRIPTION
**Pull request by 404random**
## Description
Basic functionality for ligand overlay.

## Todos
Some test cases and alignment

## Known issues
-When a carboxylic acid exists, openeye outputs as an oxt resiude, which splits the ligand resiude apart. This can be manually reformatted in the PDB file. Wil fix at later date.

-Number of confs from openbabel is not being limited correctly.

### CODE REVIEW

File: .idea/Ligandr.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maxentile) [inline .idea/Ligandr.iml]: I would add `.idea` to your `.gitignore` file, so these Pycharm files don't get included in version control

File: Ligandr/LigOverlay.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            ligand_traj = overall_dry_frame.atom_slice(overall_dry_frame.top.select(ligand))  # type: mdtraj.Trajectory
+        except IndexError as e:
+            print("Ligand selection was invalid")
+            print("Exception: %s" % str(e), file=sys.stderr)
+            traceback.print_tb(e.__traceback__, file=sys.stderr)
+            sys.exit(1)
+        if ligand_traj.n_residues > 1:
+            raise IndexError("More than one ligand residue found. Only one expected")
+        # Find approximate location of binding pocket
+        ligand_center = np.average(ligand_traj.xyz[0], axis=0) * 10
```
Comment (maxentile) [inline Ligandr/LigOverlay.py]: Could you please add a comment line about the `* 10` here? Is this a unit conversion for something later that assumes `ligand_center` is in a different unit than `ligand_traj.xyz`?

File: Ligandr/LigOverlay.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+    bound_ligand: oechem.OEGraphMol, required
+        The ligand bound to the current receptor pocket
+
+    docking_ligand: oechem.OEGraphMol, required
+        The ligand to dock.
+
+    Returns
+    -------
+    A oechem molecules with coordinates of its docking
```
Comment (maxentile) [inline Ligandr/LigOverlay.py]: Minor typo `molecules` --> `molecule`

File: Ligandr/LigOverlay.py [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+        smiles = NameToSMILES.addH(oechem.OEMolToSmiles(docking_ligand))
+        subprocess.run("obabel -:'%s' -O %s --gen3d; obabel %s -O %s --confab --conf = 100" % (
+            smiles, single_sdf.name, single_sdf.name, double_sdf.name), shell=True)
+        ins = oechem.oemolistream()
+        ins.SetConfTest(oechem.OEOmegaConfTest())
+        ins.open(double_sdf.name)
+        oechem.OEReadMolecule(ins, conformer_docking)
+
+    # Failed conformer generation
+    if conformer_docking.NumConfs() <= 1:
```
Comment (maxentile) [inline Ligandr/LigOverlay.py]: Unsure if this test should be for `<= 1` or `< 1` -- confirm?

File: Ligandr/tests/test_name_convert.py [hunk 5]
```diff
@@ -5,8 +5,6 @@
 def test_xml_grab():
     assert NameToSMILES.getSMILE("SAM") == "C[S@@+](CC[C@@H](C(=O)[O-])N)C[C@@H]1[C@H]([C@H]([C@@H](O1)n2cnc3c2ncnc3N)O)O"
 
-def test_low_pH():
-    assert NameToSMILES.addH("CC(=O)O", -2.0) == "[CH3]C(=O)[OH]\t\n"
 
 def test_high_pH():
```
Comment (maxentile) [inline Ligandr/tests/test_name_convert.py]: Name of this test may be outdated?

Comment (404random) [reply Ligandr/tests/test_name_convert.py]: I think I removed this test

File: Ligandr/LigOverlay.py (hunk 4 above)
Comment (404random) [reply Ligandr/LigOverlay.py]: It should be less than equal to one because openeye still retains a blank conformer if generation fails.

Comment (maxentile) [reply Ligandr/LigOverlay.py]: Ahh, thanks for confirming!

File: Ligandr/tests/test_name_convert.py (hunk 5 above)
Comment (maxentile) [reply Ligandr/tests/test_name_convert.py]: Sorry, I was referring to `test_high_pH()`

**Pull request merged**